pom.xml

```diff
@@ -296,21 +298,28 @@
                 <version>${slf4j.version}</version>
             </dependency>
 
+            <!-- Just to resolve the ce.kafka.version -->
```

Comment:
You might want a more detailed comment here about what is going on since it took quite a bit of discussion to figure all of this out.

Reply:
good point, will add a more detailed comment.